platform/api/_partials/resources/config/external.mdx

```diff
@@ -4,7 +4,7 @@
 
 ## `external` <span className="config-field-required" data-required="false">required</span> <span className="config-field-type">object</span> <span className="config-field-default"></span> <span className="config-field-enum"></span> <span data-pro="false" className="config-field-pro">pro</span> {#external}
 
-External holds external tool configuration
+External holds configuration for tools that are external to the vCluster.
```

Comment:
Maybe something like:

The **external** field is used to configure tools and services external to the `vCluster` deployment.